app/modules/translate.py

```
@@ -142,7 +215,30 @@ def translate():
 
     ローカルでの実行を前提としています。
     """
+    if asyncio.iscoroutinefunction(translate):
+        # 非同期環境では async_translate を実行
+        return asyncio.run(async_translate())
+    else:
+        # 同期実行
+        return _sync_translate()
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)